helm/core/values.yaml

```diff
@@ -494,7 +494,10 @@ gateway:
 
   tolerations: []
 
-  affinity: {}
+  affinity:
```

Comment:
This should cause the affinity configurations from previous versions to become ineffective, and I don't think this PR is much helpful. Users can specify their required configurations through affinity themselves